ts-rs/src/export.rs

```diff
@@ -158,9 +158,9 @@ fn export_and_merge(path: PathBuf, type_name: String, generated_type: String) ->
         .write(true)
         .open(&path)?;
 
-    let file_len = file.metadata()?.len() as usize;
+    let file_len = file.metadata()?.len();
 
-    let mut original_contents = String::with_capacity(file_len);
+    let mut original_contents = String::with_capacity(file_len as usize);
```

Comment:
This change does nothing, it was leftover from when I was testing something else